src/request.js

```diff
@@ -67,7 +67,7 @@ class Request {
   }
 
   numberBuy(first, command) {
-    let args = command.parent.rawArgs.filter(arg => (arg.indexOf('--') == -1 && arg.indexOf('nexmo') == -1 && arg.indexOf('node') == -1));
+    let args = command.parent.rawArgs.filter(arg => (arg.indexOf('-') == -1 && arg.indexOf('nexmo') == -1 && arg.indexOf('node') == -1));
     if (args.length == 2) {
       this.numberBuyFromNumber(args[1], command);
     } else if (args.length == 3) {
```

Comment:
What happens if 4 or more arguments are passed? e.g.

`nexmo sms TO FROM hello from Nexmo`

Where I forgot to quote the text.